test/suites/metrics.sh

```diff
@@ -125,6 +125,11 @@ test_metrics() {
   curl -k -s -X GET "https://${metrics_addr}/1.0/metrics" | grep "lxd_filesystem_avail_bytes" | grep "type=\"container\""
   curl -k -s -X GET "https://${metrics_addr}/1.0/metrics?project=default" | grep "lxd_filesystem_avail_bytes" | grep "type=\"container\""
 
+  # API requests metrics should be included
+  curl -k -s -X GET "https://${metrics_addr}/1.0/metrics" | grep "lxd_api_requests_completed_total" | grep "resource_group=\"metrics\""
```

Comment:
entity_type please not resource_group, lets not introduce new terminology for the same concept